test/integration_test/kubevirt_test.go

```diff
@@ -108,7 +120,14 @@ func kubevirtVMsDeployAndValidate(
 	t *testing.T,
 	instanceID string,
 	appKeys []string,
+	wg *sync.WaitGroup,
 ) []*scheduler.Context {
+	defer wg.Done()
+	// Before validating the VM create required clusterrolebindings for datavolume cloning in the VM namespace
+	testNamespace := appKeys[0] + "-" + instanceID
```

Comment:
We can't assume that there is only on app in the appKeys. Existing tests already pass multiple apps here.

Reply:
Not sure of this comment. The tests in this suite seem to be only one appkey.

Reply:
Please check all the callers of this function. e.g.
https://github.com/libopenstorage/stork/blob/28c950b28ca3a41dd7bb0a5351c1d1ba124f6877/test/integration_test/kubevirt_hyperconv_test.go#L103

Reply:
Agree, fixed this to validate VMs from all namespaces from the context not just the first one.